core/tests/protractor_utils/editor.js

```diff
@@ -142,6 +142,20 @@ var expectContentTextToEqual = function(text) {
 // Additional arguments may be sent to this function, and they will be
 // passed on to the relevant interaction editor.
 var setInteraction = function(interactionId) {
+  openInteraction(interactionId);
+  customizeInteraction.apply(null, arguments);
+  // If the "Add Response" modal opens, close it.
+  var headerElem = element(by.css(
+    '.protractor-test-add-response-modal-header'));
+  headerElem.isPresent().then(function(isVisible) {
+    if (isVisible) {
+      element(by.css('.protractor-test-close-add-response-modal')).click();
```

Comment:
This duplicates closeInteraction (or closeAddResponseModal) below. Maybe use that instead?